manifests/stable/ptp.openshift.io_ptpoperatorconfigs.yaml

```diff
@@ -56,10 +56,15 @@ spec:
                       sidecar
                     type: boolean
                   transportHost:
-                    description: 'TransportHost format is <protocol>://<transport-service>.<namespace>.svc.cluster.local:<transport-port>"
-                      Example HTTP transport: "http://ptp-event-publisher-service.openshift-ptp.svc.cluster.local:9043"
+                    description: 'TransportHost format is <protocol>://<transport-service>-<nodename>.<namespace>.svc.cluster.local:<transport-port>
```

Comment:
NODE_NAME

Reply:
removed <nodename> since the format is for both HTTP and amq. NODE_NAME is added for HTTP example below.